frontend/src/pages/SubmitProperty.test.tsx

```diff
@@ -69,22 +69,15 @@ const history = createMemoryHistory();
 const store = mockStore({
   [reducerTypes.LOOKUP_CODE]: { lookupCodes: [] },
   [reducerTypes.PARCEL]: { parcelDetail: mockDetails },
-  [reducerTypes.LEAFLET_CLICK_EVENT]: { parcelDetail: mockDetails },
-  [reducerTypes.MAP_VIEW_ZOOM]: 1,
+  [reducerTypes.LEAFLET_CLICK_EVENT]: {},
+  [reducerTypes.NETWORK]: {
+    parcel: {
+      status: 201,
+    },
+  },
 });
 
-it('SubmitProperty renders a disabled form if agency does not match current user.', () => {
-  // jest.mock('react-redux', () => ({
-  //   useDispatch: () => {},
-  //   useSelector: (state: RootState, action: IPropertyDetail) => ({
-  //     propertyDetail: {
-  //       parcelDetail: {
-  //         latitude: 48.43,
-  //         longitude: -123.37,
-  //       },
-  //     },
-  //   }),
-  // }));
+xit('SubmitProperty renders a disabled form if agency does not match current user.', () => {
```

Comment:
Is there a way to get this test working again?

Reply:
Yes, not planning on doing that for this review.